tests/insert_license_test.py

```diff
@@ -170,6 +170,7 @@ def test_insert_license_current_year_already_there(license_file_path,
                 ('module_without_license.css', '/*| *| */', 'module_with_license.css', True),
                 ('module_with_license.css', '/*| *| */', None, False),
                 ('module_with_fuzzy_matched_license.css', '/*| *| */', 'module_with_license_todo.css', True),
+                ('module_with_fuzzy_matched_license_2_asterisks.css', '/*| *| */', 'module_with_license_todo_2_asterisks.css', True),
```

Comment:
It would be interesting to also have a unit test that pass the check, don't you think?